gnosis/eth/clients/blockscout_client.py

```diff
@@ -42,6 +42,8 @@ class BlockscoutClient:
         EthereumNetwork.GODWOKEN: "https://v1.gwscan.com/",
         EthereumNetwork.VENIDIUM_TESTNET: "https://evm-testnet.venidiumexplorer.com/",
         EthereumNetwork.VENIDIUM: "https://evm.venidiumexplorer.com/",
+        EthereumNetwork.KLAYTN_TESTNET: "https://baobab.scope.klaytn.com/",
+        EthereumNetwork.KLAYTN_MAINNET: "https://scope.klaytn.com/",
```

Comment:
Is this blockexplorer blockscout? Will API work in the same way?

Reply:
Yes we have we have a testnet and mainnet. Please check https://chainlist.org/chain/8217 and https://chainlist.org/chain/1001. 
We have also deployed safe contracts and merged. https://github.com/safe-global/safe-deployments/pull/95

More References: 
**Mainnet**
Network Name: Klaytn Cypress
RPC URL: (Default: https://public-node-api.klaytnapi.com/v1/cypress)
Block Explorer URL: https://scope.klaytn.com/
Chain ID: 8217

**Testnet**
Network Name: Klaytn Baobab
RPC URL: https://api.baobab.klaytn.net:8651/
Block Explorer URL: https://baobab.scope.klaytn.com/
Chain ID: 1001 

**Documentation**
https://docs.klaytn.foundation/